packages/validator/src/validator.ts

```diff
@@ -303,9 +312,13 @@ export class Validator {
     // The total number of validators
     const total = pubkeysHex.length;
 
-    this.logger.info("Validator statuses", {...statuses, total});
+    if (metrics) {
+      for (const [status, amount] of allValidatorStatuses) {
+        metrics.validatorStatuses.inc({status}, amount);
```

Comment:
shouldn't this use `set` instead?

```suggestion
        metrics.validatorStatuses.set({status}, amount);
```

Reply:
Thanks! Good catch!